scripts/setup.ts

```diff
@@ -209,7 +209,7 @@ async function branchExists(dir: string, branch: string): Promise<boolean> {
   return exists
 }
 
-async function getVersionHashes(
+async function getPrismaCommitFromPackageJsonViaUnpkg(
```

Comment:
This rename (and signature change) might be controversial, but this is weird enough that I felt it was useful to surface this is via Pkg and essentially only to get the `prismaCommit` value.

Reply:
The function name is much more accurate 👍🏼 
And yes it's a weird part of the release code